sites/public/layouts/application.tsx

```diff
@@ -23,18 +23,7 @@ const Layout = (props) => {
       label: t(`languages.${item}`),
     })) || []
 
-  const menuLinks: MenuLink[] = [
-    {
-      title: t("nav.listings"),
-      href: "/listings",
```

Comment:
Doesn't this also remove the /listings link for all the other pages on the site? Is that going to be added somewhere else?

Reply:
Yeah it will. I think this is something to consider as well as once we add more pages where do we put those links? I think we will need to figure that out but I want to get this in for the demo tomorrow.